pkg/descriptions.json

```diff
@@ -1033,6 +1033,13 @@
         ]
     },
     {
```

Comment:
Looks like you didn't merge correctly. I suggest you go ahead and rebase to squash your changes on top of the latest master.